pytest/test_cli_features.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+import pytest
+import importlib.util
+
+def get_test_directories():
+    test_dirs = []
+    for root, dirs, files in os.walk(os.getcwd()):
+        if 'actual.py' in files and 'expected.json' in files:
+            test_dirs.append(root)
+    return test_dirs
+
+@pytest.fixture(params=get_test_directories())
+def actual_and_expected(request):
+    root = request.param
+    os.chdir(root)
+    actual_path = os.path.join(root, 'actual.py')
+    expected_path = os.path.join(root, 'expected.json')
+
+    spec = importlib.util.spec_from_file_location("actual", actual_path)
+    actual_module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(actual_module)
+    actual = actual_module.actual([], {})

```

Comment:
Just import `from gramex.install import complexity` and run `complexity()` directly.